api/edge_api/identities/views.py

```diff
@@ -314,7 +330,14 @@ def put(self, request, *args, **kwargs):
         serializer = EdgeIdentityFeatureStateSerializer(
             instance=feature_state,
             data=request.data,
-            context={"view": self, "request": request},
+            context={
```

Comment:
Is there a reason why we are not calling ` get_serializer_context` here to set the context?

Reply:
Not really, no since this view is an `APIView`, not a viewset.